website/docs/r/batch_pool.html.markdown

```diff
@@ -194,6 +196,24 @@ A `certificate` block supports the following:
 
 * `visibility` - (Optional) Which user accounts on the compute node should have access to the private data of the certificate.
 
+---
+
+A `resource_file` block supports the following:
+
+* `auto_storage_container_name` - (Optional) The storage container name in the auto storage account.
+
+* `blob_prefix` - (Optional) The blob prefix to use when downloading blobs from an Azure Storage container. Only the blobs whose names begin with the specified prefix will be downloaded. The property is valid only when `auto_storage_container_name` or `storage_container_url` is used. This prefix can be a partial filename or a subdirectory. If a prefix is not specified, all the files in the container will be downloaded.
+
+* `file_Mode` - (Optional) The file permission mode attribute in octal format. This property applies only to files being downloaded to Linux compute nodes. It will be ignored if it is specified for a `resource_file` which will be downloaded to a Windows node. If this property is not specified for a Linux node, then a default value of 0770 is applied to the file.
```

Comment:
I think it's worth calling out this is a string here, since octal format could be interpreted as an int too:


```suggestion
* `file_mode` - (Optional) The file permission mode represented as a string in octal format (e.g. `"0644"`). This property applies only to files being downloaded to Linux compute nodes. It will be ignored if it is specified for a `resource_file` which will be downloaded to a Windows node. If this property is not specified for a Linux node, then a default value of 0770 is applied to the file.
```